include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -207,6 +207,18 @@ constexpr __target __target_device =
     __target::global_buffer;
 #endif
 
+template <typename DataT>
+using __buffer_allocator =
+#if __LIBSYCL_VERSION >= 50700
+    sycl::buffer_allocator<DataT>;
```

Comment:
We have the identical definition here and on the line 218

Reply:
Fixed